src/trg-torrent-add-url-dialog.c

```diff
@@ -121,6 +131,9 @@ static void trg_torrent_add_url_dialog_init(TrgTorrentAddUrlDialog * self)
 
     w = priv->urlEntry = gtk_entry_new();
     g_signal_connect(w, "changed", G_CALLBACK(url_entry_changed), self);
+    g_signal_connect(w, "key_press_event",
```

Comment:
Use the `"activate"` signal: https://developer.gnome.org/gtk3/stable/GtkEntry.html#GtkEntry-activate

Reply:
Didn't know that it even existed. Will do. Thanks.